docs/extensions/engines/spark/lineage.md

```diff
@@ -207,3 +208,24 @@ spark.sparkContext.addSparkListener(new SparkListener {
 #### Get Lineage Events from Kyuubi EventHandler
 
 When using the `KYUUBI_EVENT` dispatcher, the lineage events will be sent to the Kyuubi `EventBus`. Refer to [Kyuubi Event Handler](../../server/events) to handle kyuubi events.
+
+#### Ingest Lineage Entities to Apache Atlas
+
+The lineage entities can be ingested into [Apache Atlas](https://atlas.apache.org/) using the `ATLAS` dispatcher.
+
+Extra works:
+
++ The least transitive dependencies needed, which are under `./extensions/spark/kyuubi-spark-lineage/target/scala-${scala.binary.version}/jars`
++ Use `spark.files` to specify the `atlas-application.properties` configuration file for Atlas
+
+Atlas Client configurations (Configure in `atlas-application.properties` or passed in `spark.atlas.` prefix):
+
+|             Name             |  Default Value  | Description                                           | Since |
+|------------------------------|-----------------|-------------------------------------------------------|-------|
+| atlas.rest.address           | localhost:21000 | The rest endpoint url for the Atlas server            | 1.8.0 |
```

Comment:
Whether the protocol should be included as prefix in rest address?

I found all the default values WITH `http` prefix in Atlas's hook is as the following
`atlas.rest.address=http://localhost:31000`

(https://github.com/apache/atlas/blob/master/addons/hive-bridge/src/test/resources/atlas-application.properties#L20)